README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+
+## ABOUT ME
+
+###Introduction
+
+My name is Kriti Bhardwaj, a second year student at Harris School of Public Policy. I am pursuing the MS in Environmental Science and Policy. I have 5 years of work experience as an environmental consultant in India.
+
+###Hobbies
+
+In my free time I love to read fiction/mythological fiction, explore architecture and write poetry.
+
+My top three favorite books are:
+
+* **The Fountainhead** by *Ayn Rand*
+* **Kane and Abel** by *Jeffrey Archer*
+* **The Alchemist** by *Paulo Coehlo*
+
+###Extra
+
+As my 30th birthday gift, my friend made a website for me https://kritibhardwaj.com, which I haven't gotten around to editing, so please don't judge! 
```

Comment:
Very cool website!  And nice inclusion of a link!

Reply:
You can format the link more cleanly using the following syntax:

```
As my 30th birthday gift, my friend made [a website for me](https://kritibhardwaj.com), which I haven't gotten around to editing, so please don't judge! 
```

As my 30th birthday gift, my friend made [a website for me](https://kritibhardwaj.com), which I haven't gotten around to editing, so please don't judge!